docstr_coverage/printers.py

```diff
@@ -147,3 +164,63 @@ def _print_overall_statistics(self, results):
         )
 
         print_line("Total coverage: {:.1f}%  -  Grade: {}".format(count.coverage(), grade))
+
+
+class JsonPrinter(Printer):
+    def print(self, results):
+        """Prints a provided set of results to stdout in format JSON.
+
+        Parameters
+        ----------
+        results: ResultCollection
+            The information about docstr presence to be printed to stdout."""
+
+        file_results, total_results = results.to_legacy()
+        print(json.dumps({"files": file_results, "result": total_results}))
+
+
+class GitHubCommentPrinter(Printer):
+    def __init__(self, verbosity: int, fail_under: int):
+        self.verbosity = verbosity
+        self.fail_under = fail_under
+
+    def print(self, results: ResultCollection):
+        """Prints a provided set of results to stdout in format JSON.
+
+        Parameters
+        ----------
+        results: ResultCollection
+            The information about docstr presence to be printed to stdout."""
+        file_results, total_results = results.to_legacy()
+
+        success = total_results["coverage"] >= self.fail_under
+
+        print_line("## [:books: docstr_coverage](https://docstr-coverage.readthedocs.io/en/latest/api_essentials.html) status: %s **%s**" % (self._status_icon(success), self._status_text(success)))
```

Comment:
We should probably link to the github readme instead of the RTD (which should be significantly improved or disabled anyways)